pages/friends/index.js

```diff
@@ -0,0 +1,86 @@
+import { useState, useEffect } from 'react'
+import axios from 'axios';
+import { Button } from '@material-ui/core'
+import { getSession } from 'next-auth/client'
+
+
+function users() {
+  const [userList, setUserList] = useState([])
+  const [friendsList, setFriendsList] = useState([])
+  const [session, setSession] = useState()
+  const [currentUser, setCurrentUser] = useState({})
+
+  useEffect(async () => {
+    const userSession = await getSession()
+    setSession(userSession)
+    if (userSession) {
+      console.log(userSession)
+      setCurrentUser({
+        'name': userSession.user.name,
+        'email': userSession.user.email
+      })
+      axios.get('/api/getuserlist', {
+        params: {
+          name: userSession.user.name,
+          email: userSession.user.email,
+        }
+      })
+        .then(res => {
+          setUserList(res.data.users)
+          setFriendsList(res.data.friends)
+        })
```

Comment:
if you're using an async function, just use await keyword. No need for a .then block. Make sure you also error handle with try catch block